src/i18n/en.json

```diff
@@ -39,6 +39,7 @@
     "button_login": "Enter",
     "enter_doppler": "Enter Doppler",
     "enter_doppler_sub": "Enjoy the benefits of Email Marketing.",
+    "error_payment": "Account blocked, please contact support.",
```

Comment:
@carodipietro this is the error message in English, let me know if you can think something better in the fly, if not you can take time and change it later.